### DISCUSSION TITLE
Creation project for GCP

### DISCUSSION THREAD

**Discussion by lisicakitsune** [vishnu-ks/Cloud-migration#209] [Q&A]
Hello! Could you help me with the following?
I am trying to create a project to migrate to GCP. I enter the subscription ID in the field, select the json file of the key created for the service account that has the owner role on this subscription, after which I press Verify and see an endless loading, while in the logs only: [ INFO: 172.18.0.1: 0 - "POST / locations HTTP/1.1" 422 Unprocessable Entity] and no more information that could help figure out what the reason is. 
Can you point out a possible error when entering data, or can you draw up a manipulation plan on the GCP side of the subscription?
Thanks!

**Comment by WeepingClown13** [comment]
Hello @lisicakitsune. Sorry that it has taken so long for this to be answered but this was a data validation bug and has been fixed. If you use the latest release/code, you will not encounter this issue.
